src/controllers/lineController.ts

```diff
@@ -1,29 +1,45 @@
 import { Application, NextFunction, Request, Response, Router } from "express";
-import { linePath } from "../consts";
+import HttpStatus from "http-status-codes";
+import { linesPath } from "../consts";
 import BadRequestException from "../exceptions/badRequestException";
 import LineService from "../services/lineService";
+import { sendResult } from "./controllerUtils";
 
-// /${prefix}/${apiVersion}/lines
+/**
+ * Controller for /${prefix}/${apiVersion}/lines
+ *
+ * @class LineController
+ */
 class LineController {
 
     private readonly _service: LineService;
 
+    /**
+     * Creates an instance of LineController.
+     * @param {Application} app
+     * @param {LineService} service
+     * @memberof LineController
+     */
     constructor(app: Application, service: LineService) {
         this.initializeRouter(app);
         this._service = service;
     }
 
-    // get(`/`)
+    /**
+     * Route for GET /
+     */
     public async getAllLines(req: Request, res: Response, next: NextFunction): Promise<void> {
         try {
             const allLines = await this._service.getAllLines();
```

Comment:
getAllLines for what? each story has it's own line - you have only one story in your app?

Reply:
For stats etc...